boltdesignsystem/bolt
# DS-157 add button element

## Jira

https://pegadigitalit.atlassian.net/browse/DS-157

## Summary

Introduces the second Bolt element named Button.

## Details

1. Button element is added to packages and PL docs.
2. Small updates were made to Text Link element to keep the two elements consistent.
3. Added mode specific CSS vars that will eventually replace theming.

## How to test

Run the branch locally and read through everything under PL > Elements > Button. Make sure all the docs make sense. Create a dummy page and try to use the twig include and HTML, ensure all props work as intended.
